README.md

```diff
@@ -118,7 +122,7 @@ torchrun --nproc-per-node 2 --no-python mistral-chat $M8x7B_DIR --instruct
 
 *Note*: Change `--nproc-per-node` to more GPUs if necessary (*e.g.* for 8x22B).
 
-- **Chat as Code Assistant**
+- **Chat with Codestral**
 
 To use [Codestral](https://mistral.ai/news/codestral/) as a coding assistant you can run the following command using `mistral-chat`.
```

Comment:
https://mistral.ai/news/codestral-mamba/ ?

Reply:
Should be in the section below :-)